css/properties/-moz-border-bottom-colors.json

```diff
@@ -20,16 +20,30 @@
             "edge_mobile": {
               "version_added": false
             },
-            "firefox": {
-              "version_added": "1",
-              "version_removed": "59",
-              "notes": "From Firefox 59, the property is limited to usage in chrome code only."
-            },
-            "firefox_android": {
-              "version_added": "4",
-              "version_removed": "59",
-              "notes": "From Firefox 59, the property is limited to usage in chrome code only."
-            },
+            "firefox": [
+              {
+                "version_added": "59",
+                "version_removed": "60",
+                "notes": "The property was removed completely in Firefox 60."
+              },
+              {
+                "version_added": "1",
+                "version_removed": "59",
+                "notes": "From Firefox 59, the property is limited to usage in chrome code only."
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "59",
+                "version_removed": "60",
+                "notes": "The property was removed completely in Firefox 60."
+              },
+              {
+                "version_added": "4",
+                "version_removed": "59",
+                "notes": "From Firefox 59, the property is limited to usage in chrome code only."
```

Comment:
Update the note: "In Firefox 59, the property is limited... only. It has completely be removed in Firefox 60."